script/iframe-messaging.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Add code here to send messages to the parent site for iframe integrations.
+ * To listen to these messages, static/js/iframe-common.js must be overriden
+ * and the iFrameResize onMessage function must be modified.
+ */
+
+ANSWERS.core.storage.registerListener({
```

Comment:
Outside the scope of this PR, but I might make an item to have a top-level `registerListener` on `ANSWERS`. That way, we're not adding references to implementation details like `ANSWERS.core.storage`. The cat's already out of the bag, the Theme is using that other places. But, if we have a top-level `ANSWERS` method, might save us some trouble if we ever change internal storage details.